conjure-java-core/src/main/java/com/palantir/conjure/java/types/UnionGenerator.java

```diff
@@ -910,11 +910,6 @@ private static MethodSpec createWrapperAcceptMethod(
         } else {
             methodBuilder.addStatement("return $N.$N($N)", visitor, visitMethodName, valueName);
         }
-        if (isDeprecated) {
-            methodBuilder.addAnnotation(AnnotationSpec.builder(SuppressWarnings.class)
-                    .addMember("value", "$S", "deprecation")
-                    .build());
-        }
```

Comment:
this change to remove the SuppressWarnings annotations seems independent from the plainSerDe change.  Can they be separate PRs, or are they interrelated somehow?

Reply:
Moved deprecation stuff to https://github.com/palantir/conjure-java/pull/1917